### PR TITLE
Adding line to untar gzip files along with bzip2

### PR DESCRIPTION
**Pull request by abottchen**
Needed for compatibility with 
https://github.com/npwalker/pe_metric_curl_cron_jobs/pull/45

### CODE REVIEW

File: view-in-grafana.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ done
 echo "ready"
 
 echo "Extracting data from tarballs..."
-find "$datadir" -type f -ctime -$RETENTION_DAYS -iname "*.bz2" -exec bash -c 'tar jxf "{}" -C $(dirname "{}")' \; 2>/dev/null;
+find "$datadir" -type f -ctime -$RETENTION_DAYS -iname "*.bz2" -exec bash -c 'tar zxf "{}" -C $(dirname "{}")' \; 2>/dev/null;
```
Comment (npwalker) [inline view-in-grafana.sh]: isn't the find still looking for .bz2 files?

Comment (abottchen) [reply view-in-grafana.sh]: Yep.  That was dumb.

File: view-in-grafana.sh [hunk 2]
```diff
@@ -45,7 +45,7 @@ done
 echo "ready"
 
 echo "Extracting data from tarballs..."
-find "$datadir" -type f -ctime -$RETENTION_DAYS -iname "*.bz2" -exec bash -c 'tar jxf "{}" -C $(dirname "{}")' \; 2>/dev/null;
+find "$datadir" -type f -ctime -$RETENTION_DAYS -iname "*.gz" -exec bash -c 'tar xf "{}" -C $(dirname "{}")' \; 2>/dev/null;
```
Comment (npwalker) [inline view-in-grafana.sh]: We probably shouldn't replace this though right?  We should just unzip files in either format so we should leave the bz2 line and add the gz line?

**Review by npwalker — CHANGES_REQUESTED** [review]
Also, squash your commits :)

**Review by npwalker — APPROVED** [review]

**Comment by npwalker** [comment]
@abottchen did you actually test this?

**Comment by abottchen** [comment]
@npwalker Yea.  It works for me with gz files and with a mixture of gz and bz2 files using this code.

**Pull request merged**